install-usercss/install-usercss.js

```diff
@@ -1,46 +1,25 @@
 /* global CodeMirror semverCompare closeCurrentTab messageBox download
-  $ $$ $create $createLink t prefs API getTab */
+  $ $$ $create $createLink t prefs API */
 'use strict';
 
 (() => {
-  const DUMMY_URL = 'foo:';
-
   // TODO: remove .replace(/^\?/, '') when minimum_chrome_version >= 52 (https://crbug.com/601425)
   const params = new URLSearchParams(location.search.replace(/^\?/, ''));
-  let liveReload = false;
+  const tabId = params.has('tabId') ? Number(params.get('tabId')) : -1;
+  const initialUrl = params.get('updateUrl');
+  if (!initialUrl) throw 'No updateUrl parameter';
+
   let installed = null;
   let installedDup = null;
+  let initialized = false;
+  let filePort;
```

Comment:
I guess this should be moved into livereload as `livereload.filePort`?

Reply:
Yeah.